demos-europe/demosplan-core
# F t33578 zip import

Ticket:
https://yaits.demos-deutschland.de/T33578
https://yaits.demos-deutschland.de/T33579

Description:
enables the option to use 'Beteiligungsimport' within the import section in ewm.
It uses the existing option to import xlsx files to extract statements - and additionally maps related files which are included
in the zip archive.
The support of original Statements was implemented here as well and therefore this PR depends on the updated Export
done in https://github.com/demos-europe/demosplan-core/pull/2444

### How to review/test
Checkout https://github.com/demos-europe/demosplan-core/pull/2444 or wait until its merged.
build FE.
goto regio and download 'abwägungstabelle as zip'
Change branch to this one.
Build FE.
then goto ewm and impot under the new 'Beteiligungsimport' section.
Check if the newly created statements have working file realtions under 'weitere Anhänge' and that every statement has
a original stn attachment. (either a custom one if present on export - or a auto-generated-original-pdf)
